DataFormats/HGCDigi/src/classes_def.xml

```diff
@@ -1,5 +1,9 @@
 <lcgdict>
-   <class name="HGCSample"/>
+   <class name="HGCSample" ClassVersion="11">
+    <version ClassVersion="11" checksum="2120380856"/>
+    <version ClassVersion="10" checksum="3298262857"/>
```

Comment:
was the old version really 10?
Was it ever recorded in the persisted data? (IIRC, the default class version defined internally by root is something  either 0 or 1).
IIUC, the "old" version 10 and the checksum for it here have no consequence and can be removed.

New class versions should start from 3
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning

Reply:
The old version was not recorded. I agree the versioning should start from 3 (had overlooked this before). I thought it was helpful to have the checksum for the old version of the class, but maybe it doesn't matter.

Reply:
The paragraph https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning is in many aspects outdated.

Quoting from the twiki: 
>  If a class was previously stored without a class version assigned then one should use 10 as the initial version instead of 3 because ROOT will automatically assign a version number to each unversioned class starting with 3 and increment the version number each time it finds an instance of a class with a different checksum while writing to the file.

we should start from version 10. @kpedro88 had asked I would record the checksum also of the version which is already in CMSSW:
> I thought it was helpful to have the checksum for the old version of the class, but maybe it doesn't matter.

Given this from the twiki:
> .. because ROOT will automatically assign a version number to each unversioned class starting with 3

I'll remove the checksum of the existing dataformat and add the checksum introduced by this PR versioning it to 10.